e2e_tests/aws_lambda/test_lambda_deployment.py

```diff
@@ -44,3 +44,13 @@ def test_aws_lambda_deployment(iris_clf_service):
         assert content == '[0]', "prediction result mismatch"
     finally:
         delete_deployment('lambda', deployment_name)
+
+
+def test_aws_lambda_deployment(iris_clf_service):
+    run_lambda_deployment(iris_clf_service)
+
+
+def test_aws_lambda_deployment_with_large_file(
+    iris_clf_service_with_additional_large_file
+):
+    run_lambda_deployment(iris_clf_service_with_additional_large_file)
```

Comment:
could you somehow verify that files were uploaded to s3 and downlod_extra_resources is invoked?

Reply:
hmmm, I will give it a shot

Reply:
might be best to just add a large dependency and import the library in the BentoService code and ensure there's no import error

Reply:
verifying that s3 bucket created etc should be straightforward right?

Reply:
Regardless of a large file/dependency or not, we will always have an s3 bucket created. Although maybe I can check is the file/dependency uploaded or not